validation/schema.json

```diff
@@ -300,6 +303,8 @@
     }
   },
   "required": [
+    "schema_format",
+    "schema_version",
```

Comment:
@oliverchang Knowing which schema to validate against is critical for parsers and something I think we should require for all OSV objects. This will be immensely helpful as we add new fields and data types, such as new reference types or relationship data to the OSV format.